app/lib/zeebe-api/zeebe-api.js

```diff
@@ -110,10 +110,10 @@ const CLIENT_OPTIONS_SECRETS = [
  */
 
 class ZeebeAPI {
-  constructor(fs, ZeebeNode, flags, log = createLog('app:zeebe-api')) {
+  constructor(fs, Camunda8, flags, log = createLog('app:zeebe-api')) {
```

Comment:
To consider: We call it `Camunda8` here, and we call it `CamundaClient` in tests. I propose we unify this, call it consistently `CamundaClient`. 

As the [SDK exposes types](https://www.npmjs.com/package/@camunda8/sdk) we can take advantage of that: I propose you add type annotations to the constructor:

```javascript
class ZeebeAPI {
  /**
   * @param { any } fs
   * @param { import('camunda8/sdk').Camunda8 } Camunda8
   * @param { any } flags
   * @param { any } [log]
   */
  constructor(fs, Camunda8, flags, log = createLog('app:zeebe-api')) {
```